libs/community/langchain_google_community/vision.py

```diff
@@ -0,0 +1,43 @@
+from typing import Iterator, Optional
+
+from langchain_core.document_loaders import BaseBlobParser
+from langchain_core.document_loaders.blob_loaders import Blob
+from langchain_core.documents import Document
+
+from langchain_google_community._utils import get_client_info
+
+
+class CloudImageLoader(BaseBlobParser):
+    def __init__(self, project: Optional[str] = None):
```

Comment:
For consistency with other connectors, can this variable be `project_id`

Reply:
actually, for LLM/ChatModels we use `project`, so I'd rather stick to `project` and eventually change everything else to follow the most frequently used integration, wdyt?